newIDE/app/src/EventsSheet/EventsTree/Renderers/LinkEvent.js

```diff
@@ -78,43 +80,51 @@ export default class LinkEvent extends React.Component<EventRendererProps, *> {
         className={classNames({
           [largeSelectableArea]: true,
           [largeSelectedArea]: this.props.selected,
+          [linkContainer]: true,
         })}
         style={styles.container}
       >
-        <span style={styles.title}>
-          Include events from{' '}
-          <i
+        <span
```

Comment:
Can we use  `<span style={styles.title}>` instead of creating a new span? By the way, I don't think selectableArea needs to be added here. Only add selectedArea on things that have an "onClick", i.e: that have interaction.

Reply:
yes we can, but i find it's not really visible at the first look that the event is disabled..

Your solution, using `<span style={styles.title}>` instead, looks like this:
![image](https://user-images.githubusercontent.com/8144048/52170244-3234e780-2747-11e9-907e-233f2e370a61.png)

While my proposal was looking like this:
![image](https://user-images.githubusercontent.com/8144048/52170253-67413a00-2747-11e9-9559-82707d6f008b.png)

In my opinion it's more clear if the whole text is crossed out (like it is for every other events in fact, when it's disabled, the whole text is crossed out, and you can't miss it's disabled).

I'm not familiar enough with the way the layout is built to avoid creating a new span or div with my proposal.. 

What do you think ?

Reply:
It's better indeed. Let me check again and merge if that's ok! :)

Reply:
Mmm, no sorry, I think it's valid to add it on `<span style={styles.title}>`. This span *includes* properly "Include events from" text.
In your screenshot you've added it to 
```
            <i
              className={classNames({
                [selectableArea]: true,
              })}
              onClick={this.edit}
            >
```

which is not what I was saying. Check my other comment ;)

Reply:
Though in this case, the icon color won't be changed... Let's try your solution